app/models/token.rb

```diff
@@ -21,21 +21,25 @@ def payload
                  end
   rescue JWT::ExpiredSignature => e
     Rails.logger.info(e.message, token: @token_string)
-    raise error_klass("Validation error: #{e.message}")
+    raise error_klass(e.message)
+  rescue JWT::DecodeError => e
+    raise error_klass(e.message)
```

Comment:
Is there a potential for token information to be exposed when the response is given here?

Reply:
No, none of these errors include any sensitive info. And it would be fine to include anything in the token in the response since the user has to send their token with the request.

The main concern would be something sensitive showing up in the logs, but none of the error messages generated by the jwt library seem problematic.